plutus-metatheory/src/site/404.html

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I was thinking `plutus-metatheory/site`, `src` seems like the Agda source to me.